raft-tech/TANF-app
# DEVAUTH - All-local devenv sans cloud or third-party auth workflows

## Summary of Changes

 - Enables direct frontend/backend communication sans Login.gov/Cloud.gov
 - Drives off new REACT_APP_DEVAUTH env var; no material changes to code if unset
 - Testing based on new dev user
   - Install via web: ./manage.py generate_dev_user
   - Create your own dev users as needed

## How to Test
 - Start frontend and backend locally with REACT_APP_DEVAUTH set.
 - Create the dev user in a web container shell: ./manage.py generate_dev_user
 - Open http://localhost:3000/data-files in browser.
 - Submit a data file normally.
 - Check logs and DBs to verify expected behaviors.
